bindings/gumjs/gumv8core.cpp

```diff
@@ -2166,10 +2169,11 @@ gumjs_native_function_init (Handle<Object> wrapper,
     String::Utf8Value type_utf8 (isolate, type);
     if (strcmp (*type_utf8, "...") == 0)
     {
-      if (is_variadic)
+      if (i == 0 || is_variadic)
       {
         _gum_v8_throw_ascii_literal (isolate,
-            "only one variadic marker may be specified");
+            "only one variadic marker may be specified and can "
+            "not be first argument");
```

Comment:
Perhaps slightly easier to read:
`only one variadic marker may be specified, and cannot be the first argument`

Reply:
OK, i will fix it.